js/messages.js

```diff
@@ -0,0 +1,100 @@
+import { isEscapeKey } from './util.js';
+import {closeUploadPopup } from './photo-input.js';
+
+//функция срабатывает при успешной загрузке фото
+function showSuccessMessage() {
+  closeUploadPopup(true);
+
+  const successMessage = createSuccessMessage();
+  document.body.appendChild(successMessage);
+
+  document.addEventListener('keydown', messageKeydownEscHandler);
+  document.addEventListener('click', outOfSucceessMessageClickHandler);
+}
+
+function createSuccessMessage() {
+  const successMessageTemplate = document.querySelector('#success').content.querySelector('.success');
+  const successMessage = successMessageTemplate.cloneNode(true);
+
+  //добавляем обработчик на кнопку закрытия фото
+  const successButtonClose = successMessage.querySelector('.success__button');
+  //добавляем обработчик закрытия при клике за окном
+  successButtonClose.addEventListener('click', () => buttonCloseClickHandler());
```

Comment:
```suggestion
  successButtonClose.addEventListener('click', buttonCloseClickHandler);
```

зачем создавать новую функцию, если достаточно передать ссылку на имеющуюся?